### ISSUE TITLE
Display synthetic Note for `AdjustDegreeOfSuccess` when there’s no DC

### ISSUE THREAD

**Issue by simonward** [foundryvtt/pf2e#16712]

**Comment by stwlam** [comment]
What information do you envision that isn't in the adjustment tooltip?

**Comment by simonward** [comment]
There isn't an adjustment tooltip when there's no DC.

Here's an example with Valeros rolling a Will save vs fear.

1. In the first result, there's a DC so the adjustment tooltip is available.
2. In the second, there is no DC, and thus no outcome to adjust. A fresh copy of Valeros has a Note RE that displays. However, there's a move towards removing the Notes.
3. The third example is Valeros with a refreshed copy of Bravery saving vs fear - there is no reminder to check the Bravery ability.

 I recognize the predicate makes it tricky, so I'm not sure it's practical.

<img width="292" alt="Screen Shot 2024-10-07 at 10 25 34 AM" src="https://github.com/user-attachments/assets/08c062a5-c4b4-41ad-92c6-f91c4f5d279b">

**Comment by stwlam** [comment]
What information would the "synthetic" note contain? We can't reliably turn a predicate into a meaningful sentence or two.

**Comment by stwlam** [comment]
The full item description?

**Comment by CarlosFdez** [comment]
I didn't like the removal of the notes, and in practice its led to more instances of "remember I have juggernaut" for me in cases where I was asked to do in improv save, but I was under the impression that I was one of the only ones who even had that opinion.

We can't really do much better than listing Bravery: One Degree Higher though, or adding a description field to item alteration itself. Item description is probably going to be too long.

**Comment by simonward** [comment]
Yeah, the entire description might be a bit long, although it would only displayed if the DC is missing. I like the “ Bravery: One Degree Higher”.

We would probably also need a suppressNote field in case a custom Note is required.

**Comment by CarlosFdez** [comment]
I remembered we will want proper item description tooltips eventually, so this might be a use case for it.
